codereview/scripts/build-antipatterns.ts

```
@@ -0,0 +1,382 @@
+  if (filePath.startsWith("controlplane/")) return "controlplane";
+  if (filePath.startsWith("studio/")) return "studio";
+  if (filePath.startsWith("cli/")) return "cli";
+  if (filePath.startsWith("connect/")) return "connect";
+  if (filePath.startsWith("composition/")) return "composition";
+  return "other";
+}
+
+function extractBotCategory(body: string): string {
+  // CodeRabbit uses prefixes like "_⚠️ Potential issue_", "_🛠️ Refactor suggestion_"
+  const categoryMatch = body.match(/^_([^_]+)_\s*\|\s*_([^_]+)_/);
+  if (categoryMatch) return categoryMatch[1].replace(/[⚠️🛠️💡]/g, "").trim();
```

Biome (2.4.4)

[error] A character class cannot match a character and a combining character.

(lint/suspicious/noMisleadingCharacterClass)